examples/v2/chat2.nim

```diff
@@ -36,7 +37,7 @@ const Help = """
 const
   PayloadV1* {.booldefine.} = false
   DefaultTopic* = "/waku/2/default-waku/proto"
-  DefaultContentTopic* = ContentTopic("dingpu")
+  DefaultContentTopic* = ContentTopic("/waku/2/huilong/proto")
```

Comment:
Would it make sense to expose a command line flag for content topic? Something like `content-topic:huilong`. Might be useful for testing etc

Reply:
Yup, it makes sense. Done in f8505c8.
For now it still defaults to `/waku/2/huilong/proto` - I want those participating in the testnet to be able to use `chat2` without any configuration.